src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -87,7 +88,12 @@ public void Start(Wallet wallet)
         private bool RemoteNode_MessageReceived(NeoSystem system, Message message)
         {
             if (message.Command == MessageCommand.Transaction)
-                consensus?.Tell(message.Payload);
+            {
+                Transaction tx = (Transaction)message.Payload;
+                if (tx.SystemFee > settings.MaxBlockSystemFee)
```

Comment:
Does it change anything at all? Consensus process only cares about a specific list of transactions when it expects them in a particular stage of the process. `CheckPrepareResponse` does the overall check anyway for the consensus process. Then non-consensus nodes won't notice this change at all and could still have such a transaction in their mempools.

Reply:
I see this setting (`MaxBlockSystemFee`) is only in the dBFT plugin now in C# implementation, so ordinary nodes can't do anything wrt this limit.